dpgen/generator/run.py

```diff
@@ -880,13 +901,15 @@ def post_train(iter_index, jdata, mdata):
         log_task("copied model, do not post train")
         return
     # symlink models
+    suffix = _get_model_suffix(jdata)
     for ii in range(numb_models):
-        if not jdata.get("dp_compress", False):
-            model_name = "frozen_model.pb"
-        else:
-            model_name = "frozen_model_compressed.pb"
+        model_name = "frozen_model%s" % suffix
+        if suffix == ".pb":
```

Comment:
should preserve the situation for pt. I think tf and pt share the same logic here.